origin-bridge/src/utils/attestation.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const eth = require('web3-eth')
+const Web3 = require('web3')
+const web3 = new Web3(process.env.PROVIDER_URL || 'http://localhost:8545')
```

Comment:
We should not need to make any call to the blockchain, so why do we need to instantiate a web3 object with a provider ?
How about using utils method directly ? For ex: Web3.utils.isHexStrict() and Web3.utils.soliditySha3()

This way also we won't have to set the env variable PROVIDER_URL for the bridge environment.

Reply:
Right on, thanks! Fixed.